unittests/svnn_ibc_tests.cpp

```diff
@@ -119,8 +115,8 @@ BOOST_AUTO_TEST_CASE(ibc_test) {} // TODO remove this and re-enable actual ibc_t
                   ("finality_mroot", block_3_result.finality_root)
                )
                ("qc", mvo()
-                  ("signature", block_5_result.qc_data.qc.value().active_policy_sig.sig.to_string())
-                  ("finalizers", active_finalizers_string(block_5_result)) 
+                  ("signature", block_4_result.qc_data.qc.value().active_policy_sig.sig.to_string())

```

Comment:
Can you expand a little more about why/how block_3_result and block_4_result are used to prove block_2 in the comments above?

Reply:
Thanks! Your new comment is very clear.